tools/ci_build/github/linux/build_linux_arm64_python_package.sh

```diff
@@ -44,7 +44,15 @@ if [ "$BUILD_DEVICE" == "GPU" ]; then
     #Enable CUDA and TRT EPs.
     ONNXRUNTIME_CUDA_VERSION="11.6"
     BUILD_ARGS+=("--use_cuda" "--use_tensorrt" "--cuda_version=$ONNXRUNTIME_CUDA_VERSION" "--tensorrt_home=/usr" "--cuda_home=/usr/local/cuda-$ONNXRUNTIME_CUDA_VERSION" "--cudnn_home=/usr/local/cuda-$ONNXRUNTIME_CUDA_VERSION" "--cmake_extra_defines" "CMAKE_CUDA_ARCHITECTURES=37;50;52;60;61;70;75;80")
+elif [ "$BUILD_DEVICE" == "CLOUD" ]; then
```

Comment:
Do not install python !!! The image already has python. When you install additional pythons, most likely you will mix them together.